Misc updates

wteouchicago opened the pull request:
DOC-353: added info about primary sites table
DOC-320: added link to explanation of annotations
DOC-396: added info/links about access, token
DOC-366: added info about Save As (vs Save)
DOC-400: updated home page image 

General cleanup:
- removed unnecessary periods at end of bullets
- replaced screenshot that had weird keyboard focus outline
- replaced screenshot that shows some old UI awkwardness

wteouchicago commented on docs/Data_Portal/Users_Guide/quick_start.md:
```diff
@@ -18,10 +18,10 @@ On the upper-left is the GDC Data Portal logo, which links to the home page of t
 * **Manage Sets**: allows the user to review gene and mutation sets that have been saved, upload new sets, and delete existing sets
 * **Cart**: where data files of interest can be added for download
-* **Login**: allows authentication for access to controlled access datasets
+* **Login**: allows authentication for access to [controlled access datasets](/Encyclopedia/pages/Controlled_Access/). Once authentication is successful, the eRA Commons username will be displayed in place of the "Login" button. Clicking on the username will then allow users to see which projects they have access to and to download an [authentication token](/Data/Data_Security/Data_Security/#auth ...
```
please check that I've done the links correctly.

wteouchicago replied:
also lemme know if we need more on access, etc.

clsackett replied:
Same as above - I think these needs to be: 
(/Encyclopedia/pages/Controlled_Access.md)
(/Data/Data_Security/Data_Security.md/#authentication-tokens)
(/Data_Transfer_Tool/Users_Guide/Getting_Started.md)
(/API/Users_Guide/Getting_Started.md)

@wwysoc2 can you verify?

wteouchicago replied:
I see some existing links with .md and some without. 😵 But I think I'll add them to be safe once bill confirms. Thanks for catching!

clsackett commented on docs/Data_Portal/Users_Guide/quick_start.md:
```diff
@@ -18,10 +18,10 @@ On the upper-left is the GDC Data Portal logo, which links to the home page of t
 * **Video Guides**: links to videos demonstrating the various features of the Data Portal
 * **Send Feedback**: provides information on how to send feedback to our team
-* **Browse Annotations**: links to the Annotations Browser, where the user can view and search for annotations that may be of use when analyzing GDC data
+* **Browse Annotations**: links to the Annotations Browser, where the user can view and search for [annotations](/Encyclopedia/pages/Annotations/) that may be of use when analyzing GDC data
```
I think this might need to be (/Encyclopedia/pages/Annotations.md)

wwysoc2 replied:
The *.md format seems to be what works based on what I see in the docs now.  I think without the *.md, the full URL will work (though it will be treated as an "external link").  

I'm not sure whether the "no .md" relative URLs will work when the site is built, they might and I'm curious, but it's probably not worth testing right now.  So let's move forward with the .md suffixes and we'll test this later.

clsackett reviewed (commented):
I think the links to the doc pages are missing ".md"

wteouchicago commented:
I've changed the following since last review, all in the quick start:
- added the .md and also updated the paths 
- swapped home page image and a case summary page

@wwysoc2 / @clsackett

wwysoc2 reviewed (approved):
lgtm!
